testng-junit-migrator/run-testng-junit-migration.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/bash
+
+set -e -u -o pipefail
+
+# If this is not a Maven build, exit here to skip Maven-specific steps.
+if [ ! -f pom.xml ]; then
+  exit 0
+fi
+
+# Use the Maven Wrapper executable if present.
+MVN='mvn'
+if [ -x ./mvnw ]; then
```

Comment:
We don't need to do this. We have this internally but there is a thread where someone mentioned it is not best practice.